auto_submit/lib/service/config.dart

```diff
@@ -71,7 +71,12 @@ class Config {
       headers: headers,
     );
     final list = json.decode(response.body).map((data) => (data) as Map<String, dynamic>).toList();
-    final String installatinId = list[0]['id'].toString();
+    String? installatinId;
```

Comment:
The `installationId` is expected to always have a value. Let's use `late String installationId` instead.

nit: `installatinId` -> `installationId`
nit: add type for the `list` variable in line 73.

Reply:
Sorry for the error. Fixed them.